client/src/components/NoteManagementModal/NoteManagementModal.js

```diff
@@ -0,0 +1,35 @@
+import { MdClose } from "react-icons/md";
+import { NoteManagementForm } from "./components";
+import "./noteManagementModal.css";
+
+const NoteManagementModal = ({
+  note = { _id: "", title: "", content: "" },
+  onNoteSubmit,
+  onClose,
+}) => {
+  const handleSubmit = (noteData) => {
+    if (note._id) {
+      onNoteSubmit(note._id, noteData);
```

Comment:
Instead of doing this for handling the create and update we can send onSubmit for each one instead of onNoteSubmit and you only need to send noteData to the onSubmit callback, for the note id you can use it from the note component